sqlx-core/src/sqlite/arguments.rs

```diff
@@ -61,6 +61,9 @@ impl SqliteArguments<'_> {
                 if name.starts_with('?') {
                     // parameter should have the form ?NNN
                     atoi(name[1..].as_bytes()).expect("parameter of the form ?NNN")
+                } else if name.starts_with('$') {
+                    // parameter should have the form $NNN
+                    atoi(name[1..].as_bytes()).expect("parameter of the form $NNN")
```

Comment:
We shouldn't panic here if the variable name isn't an integer.

Reply:
Agreed, thanks.